drivers/lte_link_control/lte_lc.c

```diff
@@ -453,9 +450,36 @@ int lte_lc_psm_get(int *tau, int *active_time)
 
 int lte_lc_edrx_req(bool enable)
 {
-	if (at_cmd_write(enable ? edrx_req : edrx_disable,
-			 NULL, 0, NULL) != 0) {
-		return -EIO;
+	char edrx_req[25];
+	int err, actt;
+	enum lte_lc_system_mode mode;
+
+	err = lte_lc_system_mode_get(&mode);
+	if (err) {
+		return err;
+	}
+
+	switch (mode) {
+	case LTE_LC_SYSTEM_MODE_LTEM:
+	case LTE_LC_SYSTEM_MODE_LTEM_GPS:
+		actt = 4;
+		break;
+	case LTE_LC_SYSTEM_MODE_NBIOT:
+	case LTE_LC_SYSTEM_MODE_NBIOT_GPS:
+		actt = 5;
```

Comment:
Could we add symbols for these constants to the list above (somewhere near line 47)?